tests/collapse-vsprof.rs

```diff
@@ -0,0 +1,118 @@
+mod common;
```

Comment:
Just as a sanity check — can you verify that you actually get a reasonable SVG at the end as well when you run the flamegraph CLI locally with this change on the `sample-default.txt` file? We should really have an end-to-end test from stack-collapse-input to flamegraph SVG for _every_ collapser, but that's not something that's currently in there. Would be happy for you to add the first (probably to `tests/flamegraph.rs`), but it's not a requirement. I just at least want a manual check that you get a good SVG.

Reply:
This produces a valid SVG.